Core/scripts/dirac-install.py

```diff
@@ -741,72 +1053,89 @@ def getModSource(self, release, modName, project=None):
       return S_OK((False, modTpl[0]))
     return S_OK((modTpl[0], modTpl[1]))
 
-  def getExtenalsVersion(self, release=False):
-    if 'DIRAC' not in self.__prjRelCFG:
+  def getExtenalsVersion(self, release=None):
+    """
+    It returns the version of DIRAC Externals. If it is not provided,
+    uses the default cfg
+
+    :param str release: the release version
+    """
+
+    if 'DIRAC' not in self.prjRelCFG:
       return False
     if not release:
-      release = list(self.__prjRelCFG['DIRAC'])
+      release = list(self.prjRelCFG['DIRAC'])
       release = max(release)
```

Comment:
ouf... this `max` here to choose the "highest" release is dangerous. It would work some time but it's just a bug to use it. Maybe it's (almost) never called, but it needs anyway to be fixed.
```
In [8]: max(['v7r0', 'v7r0-pre10'])
Out[8]: 'v7r0-pre10'

In [9]: max(['v7r0-pre9', 'v7r0-pre10'])
Out[9]: 'v7r0-pre9'
```

I suggest to use `distutils.LooseVersion`, which is already used in some DIRAC module.